tests/state/test_state_api.py

```diff
@@ -277,3 +278,38 @@ def test_requeue_message_with_pipeline(self, stub_broker, do_work, api_client, s
         res = api_client.get("messages/requeue/id1")
         assert res.json == {"error": "requeue message in a pipeline not supported"}
         assert res.status_code == 400
+
+    def test_get_result_message(self, stub_broker, stub_worker, api_client, state_middleware, redis_result_backend):
+        stub_broker.add_middleware(Results(backend=redis_result_backend))
```

Comment:
you could add a fixture result_middleware a bit like state_middleware don't you think ?

Reply:
yes, perfect